driver/docker-container/driver.go

```diff
@@ -66,12 +66,19 @@ func (d *Driver) create(ctx context.Context, l progress.SubLogger) error {
 		_, err = io.Copy(ioutil.Discard, rc)
 		return err
 	}); err != nil {
-		return err
+		// image pulling failed, check if it exists in local image store
+		err = l.Wrap("pulling failed. inspecting image "+imageName+" locally", func() error {
```

Comment:
please make it so that the log is only shown if local image did exist. and if it doesn't then previous pull error becomes fatal.